### PR TITLE
Clean up the code for 2020.

### PR DESCRIPTION
**Pull request by ghost**
Hello. 

Would you accept these small changes, to fix the broken link and improve compatibility with standard Emacs features, such as "fake version control".

Thank you for your package.

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -0,0 +1,2 @@
+*~
+\#*\#
```
Comment (jd) [inline .gitignore:2]: Not a fan of editor specific stuff.

**Review by jd — CHANGES_REQUESTED** [review]

File: .gitignore (hunk 1 above)
Comment (lockywolf) [reply .gitignore:2]: Isn't this whole project editor-specific? I can remove it, but isn't it an Emacs-specific extension?

Comment (jd) [reply .gitignore:2]: It is, but that does not change the fact anyone could modify Emacs to use different file suffixes for example.

Comment (lockywolf) [reply .gitignore]: Makes sense. Done.

**Pull request merged**